WasatchNET/ISpectrometer.cs

```diff
@@ -202,6 +202,8 @@ public interface ISpectrometer
         /// </remarks>
         bool areaScanEnabled { get; set; }
 
+        bool useRamanIntensityCorrection { get; set; }

```

Comment:
I might have called "ramanIntensityCorrectionEnabled" for consistency, but okay.

Reply:
I'll change it